lib/active_force/sobject.rb

```diff
@@ -34,8 +34,24 @@ class << self
       def_delegators :query, :not, :or, :where, :first, :last, :all, :find, :find!, :find_by, :find_by!, :sum, :count, :includes, :limit, :order, :select, :none
       def_delegators :mapping, :table, :table_name, :custom_table?, :mappings
 
+      def update(id, attributes)
```

Comment:
I moved these up here so I could easily create a shared private class method (viz., `prepare_for_update`) and have them close together.

It's confusing that we have a `class << self` block and also a bunch of class methods defined outside of it as well with `def self.method` declarations.  Might be better to migrate those into this block at some point.